packages/components/src/popover/index.tsx

```diff
@@ -478,6 +479,22 @@ const UnforwardedPopover = (
 	);
 };
 
+/**
+ * `PopoverSlot` is a component that renders a slot that is used by Popover for rendering content.
+ */
```

Comment:
No need to have the JSDocs here, we only need them inside the `Obect.assign` call